src/radiation/mocmc.cpp

```diff
@@ -64,18 +65,19 @@ void MOCMCInitSamples(T *rc) {
 
   // Meshblock geometry
   const auto geom = Geometry::GetCoordinateSystem(rc);
+  const parthenon::Coordinates_t &coords = pmb->coords;
```

Comment:
This line and lines like it throughout the code are required to resolve type inference for the templated functions in the new `UniformCartesian` class.